translator_runtime.py

```
@@ -935,21 +949,219 @@ def _list_rpa_files(game_dir):
+            "status": "failed",
+            "project_root": project_root,
+            "work_dir": work_dir,
+            "source_game_dir": source_game_dir,
+            "files_copied": 0,
+            "message": "source_game_dir must not contain work/game.",
+            "game_root_updated": False,
+        }
+    try:
+        os.makedirs(work_dir, exist_ok=True)
+        files_copied = _copy_game_directory(source_game_dir, target_game_dir)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)